[update-release-notes] Updated Command Output for XSIAM Event Collector (#3502)
demisto/demisto-sdk: CHANGELOG.md

* Fixes an issue in **format** command where the `-i` option included files in `.venv` directories.
* Updated the comment added to contribution PRs to old packs so it contains a link to the documentation of the **GitHub Codespaces** in xsoar.pan.dev.
* Updated GitPython version to 3.1.32.

> * Fixed an issue where **update-release-notes** was reporting incorrectly that XSIAM event collectors were available from Cortex XSOAR instead of Cortex XSIAM.

Reviewer: ```suggestion
* Fixed an issue where **update-release-notes** generated "available from Cortex XSOAR" instead of "from XSIAM" when run on XSIAM event collectors.
```

> * Fixed an issue when running the 'update-release-notes' command on the XSIAM event collector, the command output was incorrectly states that it is available from Cortex XSOAR instead of Cortex XSIAM.

Reviewer: see [here](https://confluence-dc.paloaltonetworks.com/display/DemistoContent/Changelogs)

Author: Updated, take a look :)

> * Fixed an issue when running the 'update-release-notes' command on the XSIAM event collector, the command output was incorrectly states that it is available from Cortex XSOAR instead of Cortex XSIAM.

Author: ```suggestion
* Fixed an issue where **update-release-notes** was reporting incorrectly that XSIAM event collectors were available from Cortex XSOAR instead of Cortex XSIAM.
```
